iOS_SDK/OneSignalSDK/Source/OneSignal.m

```diff
@@ -1626,6 +1630,12 @@ + (void)load {
     [OneSignalHelper registerAsUNNotificationCenterDelegate];
 }
 
++(BOOL) shouldDisableBasedOnProcessArguments {
+    if ([[[NSProcessInfo processInfo]arguments]containsObject: @"DISABLE_ONESIGNAL"]){
```

Comment:
nit, would be more readable in this format
```Objective-C
if ([NSProcessInfo.processInfo.arguments containsObject: @"DISABLE_ONESIGNAL"]) {
```

Reply:
Yes. This looks nicer